buyer/src/main/java/ru/otus/buyer/BuyerApplication.java

```diff
@@ -2,6 +2,7 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
```

Comment:
do we need it?

Reply:
Nope. Skimmed all project, refactored and removed unused imports.